dynawo/sources/Solvers/SolverSIM/DYNSolverSIM.cpp

```diff
@@ -396,6 +398,7 @@ void SolverSIM::solveWithoutStepRecalculation(double& tNxt) {
       */
       case CONV: {
         handleConvergence(redoStep);
+        skipScaling_ = true;
```

Comment:
Two points here (one important and one less important):

- You can also set it  to true when you have find an algebraic root so when the status is ROOT_ALG.

- This could be done also for the solveWithStepRecalculation implementation (even if we don't use it too much today)

Reply:
done